extensions/amp-story/1.0/amp-story-grid-layer.js

```diff
@@ -77,6 +77,34 @@ export const GRID_LAYER_TEMPLATE_CLASS_NAMES = {
   'thirds': 'i-amphtml-story-grid-template-thirds',
 };
 
+/**
+ * The attribute name for grid layer presets.
+ * @private @const {string}
+ */
+const PRESET_ATTRIBUTE_NAME = 'preset';
+
+/**
+ * @typedef {{
+ *  aspect-ratio: string,
+ * scaling-factor: ?string,
+ * }}
+ */
+export let PresetAttributes;
```

Comment:
Nit: values? The attributes are "2021-background" and "2021-foreground"

Reply:
Presets provide a set of attributes (`aspect-ratio` and `scaling-factor`) so that's what I was going for. I feel like preset values would be `2021-background` and `2021-foreground` since those are the values for the `preset` attribute. It does get confusing...

Maybe I can name them `PresetDetails` (and update this naming on the code as well)?